apps/frontend/src/app/5_pages/LeaderboardPointsPage/components/LeaderboardPointsSeason2/LeaderboardPointsSeason2.constants.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import { t } from 'i18next';
+
+import { prettyTx } from '@sovryn/ui';
+
+import { AmountRenderer } from '../../../../2_molecules/AmountRenderer/AmountRenderer';
+import { translations } from '../../../../../locales/i18n';
+
+export const CAMPAIGN_URL = 'https://wiki.sovryn.com/en/spice-campaign';
```

Comment:
Since this is the same in both season 1 and season 2, maybe you can move it to `LeaderboardPointsPage.constants.tsx`.